docs/ua/requirements/state-of-the-art.md

```diff
@@ -2,33 +2,144 @@
 
 ## Вступ
 
-_[Вступ повинен містити короткий огляд всього документу.]_
+Цей документ містить основні положення щодо аналізу предметної області обраної нами теми. Приводяться способи та методи вирішення завдання. Порівнюються вже існуючі системи аналізу медіа-контенту, робиться висновок щодо доцільності створення нової чи модифікації вже існуючої системи.
 
 ## Основні визначення
 
-_[Розділ містить визначення термінів та скорочень, які використовуються при аналізі предметної області.]_
+**Система** — це сукупність компонентів або елементів, що організовані для досягнення спільної мети.[1]
```

Comment:
we use another formatting. Example:
```**Система**[[1]](#system) - це певна множина взаємопов'язаних елементів або компонентів...``` (see https://github.com/serrhiy/database-course-work/blob/write-stakeholders-needs/docs/ua/requirements/stakeholders-needs.md?plain=1)